hotshot-query-service/src/data_source/storage/sql.rs

```diff
@@ -949,7 +954,15 @@ impl<Types: NodeType> MigrateTypes<Types> for SqlStorage {
 
             query.execute(tx.as_mut()).await;
 
-            tx.commit().await?;
+            // update migrated_rows column with the offset
+            tx.upsert(
+                "types_migration",
+                ["id", "completed", "migrated_rows"],
+                ["id"],
+                [(0_i64, false, offset)],
```

Comment:
@imabdulbasit Shouldn't this be `offset + limit` because when fetching we get the rows from `offset` until `offset + limit`?

Reply:
uh yeah, need to move the offset+ batch_size above this query

Reply:
https://github.com/EspressoSystems/espresso-network/pull/2953/commits/dfc6559d0e07a5410743e203924c1d154cfa99d0